spec/Model.spec.js

```diff
@@ -326,18 +326,42 @@ describe('Model', () => {
   })
 
   it('relatedToMany models can be removed', () => {
-    const note = store.add('notes', {
+    const note1 = store.add('notes', {
       id: 10,
       description: 'Example description'
     })
+    const note2 = store.add('notes', {
+      id: 11,
+      description: 'Another note'
+    })
     const todo = store.add('organizations', { id: 10, title: 'Buy Milk' })
 
-    todo.notes.add(note)
+    const notes = todo.notes
+    notes.add(note1)
+    notes.add(note2)
 
-    expect(todo.notes).toContain(note)
+    notes.remove(note1)
+    expect(notes).not.toContain(note1)
+    expect(notes).toContain(note2)
```

Comment:
this specifically tests the `RelatedRecordsArray` object, since we remove items from both it *and* the parent record's relationships